pkg/controller/sdk/reconciler.go

```diff
@@ -46,8 +47,8 @@ var _ reconcile.Reconciler = &Reconciler{}
 // Reconcile compares the actual state with the desired, and attempts to
 // converge the two.
 func (r *Reconciler) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-	ctx := context.TODO()
-	logger := logging.FromContext(ctx)
+	ctx := logging.WithLogger(context.TODO(), r.logger.With(zap.Any("request", request)))
+	logger := r.logger
```

Comment:
Shouldn't this be `logger := logging.FromContext(ctx)`? As-is, you are missing the `request` parameter added on the previous line.

Reply:
Fixed in commit b18e66e